jaxws-ri/extras/jaxws-maven-plugin/pom.xml

```diff
@@ -173,7 +173,7 @@
         <warn.limit>150</warn.limit>
 
 <!--        <jaxws-tools.version>${project.version}</jaxws-tools.version>-->
-        <jaxws-tools.version>3.0.2</jaxws-tools.version>
+        <jaxws-tools.version>4.0.0-SNAPSHOT</jaxws-tools.version>
```

Comment:
keep 3.0.2 here otherwise tests are going to fail

Reply:
3.0.2 is back